### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by maciejpi**
@incuna/frontend please review

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# upcoming (5.3.1)
+
+* Update README
```
Comment (wytrych) [inline CHANGELOG.md]: Not sure it's worth putting that in the changelog

Comment (maciejpi) [inline CHANGELOG.md]: yea... so shall I just push to master or here but without the CHANGELOG change?

Comment (wytrych) [inline CHANGELOG.md]: Yes, I wouldn't bother with the Changelog, but maybe there are other opinions?

**Comment by maciejpi** [comment]
I removed the note in `CHANGELOG.md`

**Pull request merged**